css-multicol-2/Overview.bs

```diff
@@ -360,11 +360,14 @@ The multi-column model</h2>
 
 	The column boxes are ordered
 	in the [=inline base direction=] of the multicol container
-	and arranged into <dfn lt="multi-column line | multi-col line | multicol line">multicol lines</dfn>.
+	and arranged into 
```

Comment:
The column boxes are arranged into lines that are inside a row, no?

Reply:
I'm not so sure conceptually, because the height of the column (`column-height`) is what defines the size of the row. It could, however, contain two lines and a spanner.

Reply:
Yes, a row contains lines and/or spanners. Lines contain columns. So the column boxes are arranged into lines whose height are confined by the containing row, due to `column-height`.

So I was thinking that this part looked right the way it was.

> The column boxes are ordered in the inline base direction of the multicol container and arranged into multicol lines.

*Then* explain that a line is contained by a row?

Reply:
I've tried to do this this way round PTAL.